src/Kernel/Constants/Rights.cs

```diff
@@ -2,19 +2,21 @@
 {
   /// <summary>
   /// Rights give access to admin rights to the corresponding services or blocks
-  /// Admin, Feddback, Email, create/update Roles only work with admin rights
+  /// Admin, Feedback, Email, create/update Roles only work with admin rights
   /// </summary>
-  /// <param name="AddEditRemoveUsers">Right grants admin rights to user, family, education, skill, office users, achievement </param>
-  /// <param name="AddEditRemoveProjects">Right grants admin rights to project, task services</param>
-  /// <param name="AddEditRemoveEmailsTemplates">Right grants admin rights to text template service</param>
-  /// <param name="AddEditRemoveDepartments">Right grants admin rights to department service, work with project department</param> 
-  /// <param name="AddEditRemoveNews">Right grants admin rights to news service</param> 
-  /// <param name="AddEditRemovePositions">Right grants admin rights to position service</param> 
-  /// <param name="AddEditRemoveTime">Right grants admin rights to time service</param> 
-  /// <param name="AddEditRemoveCompanies">Right grants admin rights to company, office services</param> 
-  /// <param name="AddEditRemoveCompanyData">Right grants admin rights to company service to work with company users, office workspaces/workspaceTypes, booking service</param> 
-  /// <param name="AddRemoveUsersRoles">Right grants admin rights to right service to work with users roles</param> 
-  /// <param name="AddEditRemoveWiki">Right grants admin rights to wiki service</param>
+  /// <value name="AddEditRemoveUsers">Right grants admin rights to user, family, education, skill, office users, achievement </value>
+  /// <value name="AddEditRemoveProjects">Right grants admin rights to project, task services</value>
+  /// <value name="AddEditRemoveEmailsTemplates">Right grants admin rights to text template service</value>
+  /// <value name="AddEditRemoveDepartments">Right grants admin rights to department service, work with project department</value>
+  /// <value name="AddEditRemoveNews">Right grants admin rights to news service</value>
+  /// <value name="AddEditRemovePositions">Right grants admin rights to position service</value>
+  /// <value name="AddEditRemoveTime">Right grants admin rights to time service</value>
+  /// <value name="AddEditRemoveOffices">Right grants admin rights to office service</value>
+  /// <value name="AddEditRemoveCompanies">Right grants admin rights to company service</value>
+  /// <value name="AddEditRemoveBooking">Right grants admin rights to booking service</value>
+  /// <value name="AddEditRemoveCompanyData">Right grants admin rights to company service</value>
```

Comment:
Right grants admin rights to company service to work with company users

AddEditRemoveCompanies!=AddEditRemoveCompanyData